packages/cells/src/resizeHandle.ts

```diff
@@ -93,4 +74,40 @@ export class ResizeHandle extends Widget {
         break;
     }
   }
+
+  /**
+   * Handle `after-attach` messages.
+   */
+  protected onAfterAttach(msg: Message) {
+    this.node.addEventListener('dblclick', this);
+    this.node.addEventListener('mousedown', this);
+    super.onAfterAttach(msg);
+  }
+
+  /**
+   * Handle `before-detach` messages.
+   */
+  protected onBeforeDetach(msg: Message) {
+    this.node.removeEventListener('dblclick', this);
+    this.node.removeEventListener('mousedown', this);
+    super.onBeforeDetach(msg);
+  }
+
+  private _resize(event: MouseEvent): void {
+    // Gate the output size ratio between {0.05, 50} as sensible defaults.
+    const { width, x } = this.targetNode.getBoundingClientRect();
+    const position = event.clientX - x;
+    const ratio = width / position - 1;
+    if (0 < ratio) {
+      const normalized = Math.max(Math.min(Math.abs(ratio), 50), 0.05);
```

Comment:
Seems that this is more than enough:

![limits](https://user-images.githubusercontent.com/5832902/235912594-c9f28d08-af21-4026-8963-16f7aa816f23.gif)

Reply:
The grid system has a native [`minmax`](https://developer.mozilla.org/en-US/docs/Web/CSS/minmax) function for this use case, but I recall it may require some experimentation to get this right. I was just documenting the current behaviour as of the state of this PR. I believe that we can merge this PR and improve later (would be good to open an issue targeting 4.0.x).

Reply:
(and maybe a change is needed in another part altogether)

Reply:
I opened #14485, merging as-is.